test/routing/VersionRoutingMapSpec.scala

```diff
@@ -45,7 +47,20 @@ class VersionRoutingMapSpec extends UnitSpec with GuiceOneAppPerSuite {
         val versionRoutingMap: VersionRoutingMapImpl = VersionRoutingMapImpl(
           defaultRouter = defaultRouter,
           v4Router = v4Routes,
-          v5Router = v5Routes
+          v5Router = v5Routes,
+          v6Router = v6Routes
+        )
+
+        versionRoutingMap.map(Version5) shouldBe v5Routes
+      }
+
+      s"route to ${v6Routes.toString}" in {
+
+        val versionRoutingMap: VersionRoutingMapImpl = VersionRoutingMapImpl(
+          defaultRouter = defaultRouter,
+          v4Router = v4Routes,
+          v5Router = v5Routes,
+          v6Router = v6Routes
         )
 
         versionRoutingMap.map(Version5) shouldBe v5Routes
```

Comment:
this line should be:
```suggestion
        versionRoutingMap.map(Version6) shouldBe v6Routes
```